static/js/shared/stats_var_titles.ts

```diff
@@ -45,9 +45,23 @@ const STATS_VAR_TITLES: { [key: string]: string } = {
     "Gross National Income (Purchasing Power Parity) Per Capita",
   Amount_EconomicActivity_GrossNationalIncome_PurchasingPowerParity:
     "Gross National Income (Purchasing Power Parity)",
+  "WorldBank/SL_TLF_0714_ZS": "Children Ages 7-14 in Employment",
```

Comment:
Should I revert these changes, since there is no ranking for int'l vars right now?

Reply:
please keep it

Reply:
Ack